packages/cli/commands/executeBatch.ts

```diff
@@ -0,0 +1,860 @@
+import * as fs from 'fs';
+import { 
+	Command, 
+	flags,
+} from '@oclif/command';
+
+import {
+	UserSettings,
+} from 'n8n-core';
+
+import {
+	IDataObject,
+	INode, 
+	INodeExecutionData, 
+	IRun, 
+	ITaskData, 
+} from 'n8n-workflow';
+
+import {
+	ActiveExecutions,
+	CredentialsOverwrites,
+	CredentialTypes,
+	Db,
+	ExternalHooks,
+	IExecutionsCurrentSummary,
+	IWorkflowDb,
+	IWorkflowExecutionDataProcess,
+	LoadNodesAndCredentials,
+	NodeTypes,
+	WorkflowCredentials,
+	WorkflowRunner,
+} from "../src";
+
+import { 
+	sep 
+}  from 'path';
+
+import { 
+	diff,
+} from 'json-diff';
+import { 
+	ObjectID,
+} from 'typeorm';
+
+import { 
+	getLogger,
+} from '../src/Logger';
+
+import {
+	LoggerProxy,
+} from 'n8n-workflow';
+
+const colorOutput = true;
+const executionTimeout = 3 * 60 * 1000;
```

Comment:
This was added as a placeholder so it could become configurable or changed in the future. No reason for it not being static or any other option.

Thoughts? Any reasons about changing it to static?

Reply:
I was mostly curious about the reasoning behind the difference.

Personally I would place them among static fields if they serve the same purpose - if we want to make them configurable, I would make an option readable from a config file or the like.